src/routes/(console)/project-[region]-[project]/messaging/providers/table.svelte

```diff
@@ -39,51 +39,68 @@
     }
 </script>
 
-<MultiSelectionTable
-    resource="provider"
-    columns={$columns}
-    onDelete={handleDelete}
-    allowSelection={$canWriteProviders}>
-    {#snippet header(root)}
-        {#each $columns as { id, title }}
-            <Table.Header.Cell column={id} {root}>{title}</Table.Header.Cell>
-        {/each}
-    {/snippet}
+<div class="table-scroll-wrapper">
```

Comment:
we don't need to add custom classes here, best to add `width: { min: xxx }` on the columns itself. You can easily search for the most commonly used dimensions.
Same for table on Overview as well 👍